src/view_page.cpp

```diff
@@ -555,23 +554,20 @@ void View::DrawBrace(DeviceContext *dc, int x, int y1, int y2, int staffSize)
     assert(dc);
     if (m_doc->GetOptions()->m_useBraceGlyph.GetValue()) {
         Glyph *glyph = Resources::GetGlyph(SMUFL_E000_brace);
-        if (glyph) {
-            FontInfo* font = m_doc->GetDrawingSmuflFont(staffSize, false);
-            int dummy, g_w, g_h;
-            glyph->GetBoundingBox(dummy, dummy, g_w, g_h);
-            const float ratio = static_cast<float>(glyph->GetUnitsPerEm()) / font->GetPointSize();
-            const float scale = static_cast<float>(y1 - y2) / g_h;
-            int braceWidth =  m_doc->GetDrawingDoubleUnit(staffSize);
-            x -= braceWidth + m_doc->GetDrawingBeamWhiteWidth(staffSize, false) / 2 + m_doc->GetDrawingUnit(staffSize);
-            const float currentWidthToHeightRatio = font->GetWidthToHeightRatio();
-            const float widthAfterScalling = g_w * scale;
-            font->SetWidthToHeightRatio(static_cast<float>(braceWidth) / widthAfterScalling);
-            DrawSmuflCode(dc, x, y2, SMUFL_E000_brace, staffSize * ratio * scale, false);
-            font->SetWidthToHeightRatio(currentWidthToHeightRatio);
-            return;
-        } else {
-            LogError("Cant find brace glyph(SMUFL_E000_brace). Use the standart brace pattern.");
-        }
+        assert(glyph);
+        FontInfo* font = m_doc->GetDrawingSmuflFont(staffSize, false);
+        int dummy, g_w, g_h;
```

Comment:
Doc::GetGlyphHeight does not work for this?

Reply:
No. It performs some calculations which are not needed.

Reply:
Could you be more specific? You should not have to call glyph->GetUnitsPerEm from the View. Also, since the SMuFL spec are explicit about the size of the brace glyph (i.e., the height of a single five-line stave) you should not have to retrieve it.